docs-content/installing.html.md.erb

```diff
@@ -31,64 +31,93 @@ This adds the tile to your staging area.
 
 1. Use `cf marketplace` to verify availability of the community service named `cyberark-conjur`.
 
-###<a id='assign-az'></a>Assign AZs and Networks
+### <a id='assign-az'></a>Assign AZs and Networks
 
 Choose appropriate values to configure where to deploy CyberArk Conjur Service Broker for PCF.
 
-###<a id='config-service-broker'></a>Service Broker Configuration
+### <a id='config-service-broker'></a>Service Broker Configuration
 
 This tab configures communication between the Service Broker and a Conjur appliance.
 
-![Conjur Service Broker Configuration tab](service_broker_config_screen.png)
+![Conjur Service Broker Configuration tab](tile_config_screen.png)
```

Comment:
this is going to need to be updated now that we don't have the "preserve policy" checkbox. it's just a screenshot of the tile config screen.